Sources/SwiftDocC/LinkTargets/LinkDestinationSummary.swift

```diff
@@ -254,23 +258,94 @@ extension LinkDestinationSummary {
     ///   - path: The bundle-relative path to this page.
     ///   - taskGroups: The task groups that lists the children of this page.
     ///   - compiler: The content compiler that's used to render the node's abstract.
-    init(documentationNode: DocumentationNode, path: String, taskGroups: [TaskGroup], platforms: [PlatformAvailability]?, compiler: inout RenderContentCompiler) {
-        let symbol = documentationNode.semantic as? Symbol
+    init(documentationNode: DocumentationNode, path: String, taskGroups: [TaskGroup], taskGroupVariants: [[RenderNode.Variant.Trait]: [TaskGroup]], platforms: [PlatformAvailability]?, compiler: inout RenderContentCompiler) {
+        let redirects = (documentationNode.semantic as? Redirected)?.redirects?.map { $0.oldPath }
+        let referenceURL = documentationNode.reference.url
+        
+        guard let symbol = documentationNode.semantic as? Symbol, let summaryTrait = documentationNode.availableVariantTraits.first(where: { $0.interfaceLanguage == documentationNode.sourceLanguage.id }) else {
```

Comment:
Should we have a feature flag check here?

Reply:
Discussed this with @d-ronnqvist offline: This isn't guarding on multi-language versus non-multi-language as I was originally thinking. It's guarding symbol vs non-symbol.

The feature flag shouldn't be used here and we can rely on feature flags further up in the compilation stack to prevent mixed-language symbols from being vended to this function if we haven't enabled objective-c support.